<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>297</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>631</width>
    <height>297</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>631</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cylinder Volume Calculator</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <column>
    <property name="text">
     <string>Index</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radius</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radius Unit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Height</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Height Unit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Volume</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A&amp;dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>De&amp;lete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApplyDelete">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>245</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblErrorMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVolume">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>211</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Volume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalVolume">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>211</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
